### ISSUE TITLE
Incompatibility with `nova-flexible-content` field

### ISSUE THREAD

**Issue by milewski** [BBS-Lab/nova-file-manager#70]
Hi, version [0.5.0](https://github.com/BBS-Lab/nova-file-manager/releases/tag/v0.5.0) introduced a change in the way it fetches the list of files on the modal window that breaks compatibility with `flexible-content-fields` package. 

Here on this function, it tries to find the `FileManager` field by instance, but when using it inside a flexible layout it throws a 404 error that prevents the window from opening... 

https://github.com/BBS-Lab/nova-file-manager/blob/d499da0d95e03e2e1b6ebc2fc9156dcc251c0de4/src/Http/Requests/BaseRequest.php#L50-L59

**Comment by milewski** [comment]
Workaround:

```php
class Page extends Resource
{
    public function fields(NovaRequest $request): array
    {
        $fields = [];

        if ($request instanceof \BBSLab\NovaFileManager\Http\Requests\IndexRequest) {
            $fields[] = FixedFileManager::make($request->input('attribute'));
        }

        return [

            Flexible::make('Components', 'components')
                ->addLayout('...many layouts...'),

            ...$fields,

        ];
    }
}
```


````php
class FixedFileManager extends FileManager
{
    protected function resolveAttribute($resource, $attribute = null): ?array
    {
        ...
        if (is_array($value)) {
            if (Arr::isList($value)) {
                $value = collect($value)->map(fn(array $asset) => new Asset(...$asset));
            } else {
                $value = collect([ new Asset(...$value) ]);
            }
        }
        ...      
    }
}
```

**Comment by mikaelpopowicz** [comment]
Hi

Thank you for reporting.

The flexible content support was released in the 0.6.0.
